src/ui/wxWidgets/PasswordSafeFrame.cpp

```diff
@@ -2750,6 +2777,7 @@ void PasswordSafeFrame::UnlockSafe(bool restoreUI, bool iconizeOnCancel)
   }
 
   CreateMenubar(); // Recreate menubar to replace menu item 'Unlock' by 'Lock'
+  RefreshToolbarButtons();
```

Comment:
It seems strange to me that we have to refresh the toolbar right after creating it. Why can't create take into account the lock status?

Reply:
_CreateMenubar_ creates/recreates the menu items of the menu bar, while _RefreshToolbarButtons_ updates the toolbar items. With the changes, not only the menu is now updated, but also the toolbar. (Please see also my [comment](https://github.com/pwsafe/pwsafe/pull/1501#discussion_r2085372554) above.)